target/linux/rtl838x/image/Makefile

```diff
@@ -0,0 +1,44 @@
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+include $(TOPDIR)/rules.mk
+include $(INCLUDE_DIR)/image.mk
+
+KERNEL_LOADADDR = 0x80000000
+KERNEL_ENTRY = 0x80000400
+
+define Device/Default
+  PROFILES = Default
+  KERNEL := kernel-bin | append-dtb | gzip | uImage gzip
+  KERNEL_INITRAMFS := kernel-bin | append-dtb | gzip | uImage gzip
+  DEVICE_DTS_DIR := ../dts
+  DEVICE_DTS = $$(SOC)_$(1)
+  IMAGES := sysupgrade.bin
+  SUPPORTED_DEVICES := $(subst _,$(comma),$(1))
+  sysupgrade_bin := append-kernel | pad-to 64k | append-rootfs | pad-rootfs
+  IMAGE/sysupgrade.bin := append-kernel | pad-to 64k | append-rootfs | pad-rootfs | append-metadata | check-size
```

Comment:
Please wrap after 74 or 80 characters per line:

```
  IMAGE/sysupgrade.bin := append-kernel | pad-to 64k | append-rootfs | \
	pad-rootfs | append-metadata | check-size
```

Hanging indent with one single tab in this case (Device nodes).

Reply:
Done